shipper.py

```diff
@@ -507,6 +515,10 @@ def session_get_user(cookie, my_db):
     else:
         return None
 
+def addDataPath(path):
+    return('data/' +path)
```

Comment:
Never concatenate paths as Strings, see `os.path.join()` function, which is more suitable here.